reg/templates/reg/register.html

```diff
@@ -0,0 +1,15 @@
+
+<h3>REGISTER</h3>
+<hr>
+
+<form action="" method="POST">
+    {% csrf_token %}
+
+    {% for field in form %}
+        <div>
+            <p>{{ field.label }}: <br> {{ field }}</p> 
+
+        </div>
+    {% endfor %}
```

Comment:
テキストボックスのラベルとテキストボックスを`for~endfor`で出力するだけだと、エラーメッセージがうまく出てきません（パスワードが短い・一般的すぎなど）。ここは以下のように訂正します。
```suggestion
<form action="" method="POST">
    {% csrf_token %}
    {{ form.as_p }}
```